src/main/java/seedu/mindmymoney/helper/Calculations.java

```diff
@@ -35,7 +45,68 @@ public static void calculateExpenditurePerMonth(String input, ExpenditureList ex
         if (sumOfExpenditure == 0.0) {
             throw new MindMyMoneyException("Month and year not found in the list! Do check your input");
         }
-        System.out.println("Total expenditure in the month of " + input + " is " + sumOfExpenditure);
+        System.out.println("Total expenditure in the month of " + input + " is $" + sumOfExpenditure + ".");
+        displayExpenditureBreakdown(foundItems, sumOfExpenditure);
     }
 
+    /**
+     * Displays the expenditure breakdown for the given month.
+     *
+     * @param foundItems The list containing all the expenses in the month.
+     * @param sumOfExpenditure Total sum of expenses in the month.
+     */
+    public static void displayExpenditureBreakdown(ArrayList<Expenditure> foundItems, float sumOfExpenditure) {
+        float foodPercentage = calculatePercentage(FOOD, foundItems, sumOfExpenditure);
+        float transportPercentage = calculatePercentage(TRANSPORT, foundItems, sumOfExpenditure);
+        float utilitiesPercentage = calculatePercentage(UTILITIES, foundItems, sumOfExpenditure);
+        float personalPercentage = calculatePercentage(PERSONAL, foundItems, sumOfExpenditure);
+        float entertainmentPercentage = calculatePercentage(ENTERTAINMENT, foundItems, sumOfExpenditure);
+        float othersPercentage = calculatePercentage(OTHERS, foundItems, sumOfExpenditure);
+
+        System.out.println(System.lineSeparator() + "BREAKDOWN OF EXPENSES:");
+        System.out.print(PrintStrings.LINE);
+        System.out.println("FOOD:          " + printBar(foodPercentage) + " " + foodPercentage + "%");
+        System.out.println("TRANSPORT:     " + printBar(transportPercentage) + " " + transportPercentage + "%");
+        System.out.println("UTILITIES:     " + printBar(utilitiesPercentage) + " " + utilitiesPercentage + "%");
+        System.out.println("PERSONAL:      " + printBar(personalPercentage) + " " + personalPercentage + "%");
+        System.out.println("ENTERTAINMENT: " + printBar(entertainmentPercentage) + " " + entertainmentPercentage + "%");
+        System.out.println("OTHERS:        " + printBar(othersPercentage) + " " + othersPercentage + "%");
+        System.out.println(PrintStrings.LINE);
+    }
+
+    /**
+     * Displays the chart for each category type.
+     *
+     * @param percentage Percentage of each category type that is part of the expenses.
+     * @return The 'bar' chart to be output for the category type.
+     */
+    public static String printBar(float percentage) {
+        String output = "";
+        for (float i = 0; i < percentage / 10; i += 0.5) {
```

Comment:
can explain abit into what this funciton does, like putting the magic numbers into constants

Reply:
Resolved